aws-geo-location/src/main/java/com/amplifyframework/geo/location/AWSLocationGeoPlugin.kt

```diff
@@ -64,6 +64,7 @@ class AWSLocationGeoPlugin(
         configuration.searchIndices!!.default
     }
 
+    @Deprecated("This credentialsProvider was released with public visibility, but is not intended to be consumed.")
```

Comment:
The Geo credentialsProvider is used in maplibre-adapter [here](https://github.com/aws-amplify/amplify-android/blob/main/maplibre-adapter/src/main/java/com/amplifyframework/geo/maplibre/http/AWSRequestSignerInterceptor.kt#L105). Could we use a similar strategy as in other parts of this PR for showing a warning message to customers if they use this variable while still allowing use within the library?

Reply:
Nice catch. Applied.